### PR TITLE
Add Consul Web UI - v0.5.0

### PR DESCRIPTION
**Pull request by housepage**

### CODE REVIEW

**Comment by fanquake** [comment]
Can you please squash your commits.

File: Casks/consul-web-ui.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version '0.5.0'
+  sha256 '0081d08be9c0b1172939e92af5a7cf9ba4f90e54fae24a353299503b24bb8be9'
+
+  # bintray.com is the official download host per the vendor homepage
+  url "https://dl.bintray.com/mitchellh/consul/#{version}_web_ui.zip"
+  name 'Consul - Web UI'
+  homepage 'http://www.consul.io/'
+  license :mpl
+
+  artifact 'dist/index.html'
```
Comment (vitorgalvao) [inline Casks/consul-web-ui.rb]: A `:target` needs to be set.

**Comment by vitorgalvao** [comment]
@housepage Sill planning to work on this?

**Comment by vitorgalvao** [comment]
Done by another user.

**Pull request closed without merging**